test/e2e/flowaggregator_test.go

```diff
@@ -427,6 +426,26 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 		}
 	})
 
+	// ToExternalFlows tests the export of IPFIX flow records when a source Pod
+	// sends traffic to an external IP
+	t.Run("ToExternalFlows", func(t *testing.T) {
+		// Creating a agnhost server as a host network pod
+		serverPodPort := int32(80)
+		_, serverIPs, cleanupFunc := createAndWaitForPod(t, data, func(name string, nodeName string) error {
+			return data.createServerPod(name, "", serverPodPort, false, true)
+		}, "test-server-", controlPlaneNodeName())
+		defer cleanupFunc()
+
+		clientName, clientIPs, cleanupFunc := createAndWaitForPod(t, data, data.createBusyboxPodOnNode, "test-client-", controlPlaneNodeName())
+		defer cleanupFunc()
```

Comment:
if the fact that it is a control-plane Node is irrelevant to the test, it's better to use `nodeName(0)`

Reply:
I use `controlPlaneNodeName()` because I found it is used at many places in this file. I wanted to follow the same format. Can I know why it's better to use `nodeName(0)`?

Reply:
because the fact that it is a control-plane Node is irrelevant. You just need a given Node, and since you don't know the size of the cluster on which the test is running and all you need is a single Node, you pick the first one, which is `nodeName(0)`. Based on our framework, `nodeName(0)` is actually the control-plane Node, but it doesn't have to be this way in the future.

Reply:
got it. will update. thanks!